### PR TITLE
Max/float53

### PR DESCRIPTION
**Pull request by max-hoffman**
Follow specific rules for mysql float/double/error depending on the float ops.

### CODE REVIEW

File: sql/errors.go [hunk 1]
```diff
@@ -1,4 +1,4 @@
-// Copyright 2020-2021 Dolthub, Inc.
+// Copyright 2021 Dolthub, Inc.
```
Comment (Hydrocharged) [inline sql/errors.go:1]: We should probably keep the `2020` portion on this file.

File: sql/type.go [hunk 2]
```diff
@@ -1,4 +1,4 @@
-// Copyright 2020-2021 Dolthub, Inc.
+// Copyright 2021 Dolthub, Inc.
```
Comment (Hydrocharged) [inline sql/type.go:1]: Same comment about `2020`

File: sql/errors.go [hunk 3]
```diff
@@ -130,6 +130,9 @@ var (
 	// ErrTruncateReferencedFromForeignKey is returned when a table is referenced in a foreign key and TRUNCATE is called on it.
 	ErrTruncateReferencedFromForeignKey = errors.NewKind("cannot truncate table %s as it is referenced in foreign key %s on table %s")
 
+	// ErrInvalidPrecisionSyntax is returned when a column type-definition has argument violations.
+	ErrInvalidPrecisionSyntax = errors.NewKind("column %s type definition is invalid: %s")
```
Comment (Hydrocharged) [inline sql/errors.go:134]: The error variable name is more specific than the error message. I'd rename it to `ErrInvalidColTypeDefinition` based on the error message.

**Review by Hydrocharged — APPROVED** [review]
Overall looks good! Just a few small comments and it's ready to merge!

File: sql/errors.go (hunk 3 above)
Comment (max-hoffman) [reply sql/errors.go]: 👍 Yeah makes sense

**Pull request merged**